frontend/testing/playwright/tests/org-library/codelists.spec.ts

```diff
@@ -57,34 +57,19 @@ test('that it is possible to create a new codelist', async ({ page, testAppName
   // This situation might happen if a test is cancelled right after the creation of the code list.
   if (codeListTitleAlreadyExists) {
     await orgLibraryPage.codeLists.clickOnCodeListAccordion(CODELIST_TITLE_MANUALLY);
-    await deleteAndVerifyDeletionOfCodeList(orgLibraryPage, CODELIST_TITLE_MANUALLY);
+    await deleteAndVerifyDeletionOfOpenedCodeList(orgLibraryPage, CODELIST_TITLE_MANUALLY);
   }
 
   await orgLibraryPage.codeLists.clickOnCreateNewCodelistButton();
   await orgLibraryPage.codeLists.verifyNewCodelistModalIsOpen();
   await orgLibraryPage.codeLists.writeCodelistTitle(CODELIST_TITLE_MANUALLY);
 
-  await orgLibraryPage.codeLists.clickOnAddAlternativeButton();
-
-  await orgLibraryPage.codeLists.verifyNewItemValueFieldIsVisible(
-    EXPECTED_NUMBER_OF_ITEMS_IN_MANUALLY_CREATED_CODELIST,
-  );
-  const firstRowValue: string = 'First value';
-  await orgLibraryPage.codeLists.writeCodelistValue(
-    EXPECTED_NUMBER_OF_ITEMS_IN_MANUALLY_CREATED_CODELIST,
-    firstRowValue,
-  );
-  const firstRowLabel: string = 'First label';
-  await orgLibraryPage.codeLists.writeCodelistLabel(
-    EXPECTED_NUMBER_OF_ITEMS_IN_MANUALLY_CREATED_CODELIST,
-    firstRowLabel,
-  );
-
   await orgLibraryPage.codeLists.clickOnSaveCodelistButton();
   await orgLibraryPage.codeLists.verifyThatCodeListIsVisible(CODELIST_TITLE_MANUALLY);
 });
 
-test('that it is possible to add a new row to an existing codelist and modify the fields in the row', async ({
+// Todo: Remove "skip" and adjust this test when creation of text resources is implemented: https://github.com/Altinn/altinn-studio/issues/15048
+test.skip('that it is possible to add a new row to an existing codelist and modify the fields in the row', async ({
```

Comment:
There is no reasonable way to test this without any possibility to add a text resource to test it on. While I'm writing this, there _are_ text resources to choose from, but that's a mock that's going to be removed.